src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -37,6 +37,7 @@
 
         <service id="validator.mapping.cache.apc" class="%validator.mapping.cache.apc.class%" public="false">
             <argument>%validator.mapping.cache.prefix%</argument>
+            <deprecated>The "%service_id%" service is deprecated since Symfony 2.8 and will be removed in 3.0.</deprecated>
```

Comment:
Is it Symfony 2.8 or 2.5 as described by @WouterJ in https://github.com/symfony/symfony/pull/16865/files#diff-c2459e2c1a786adcaaab00b70630b67fR39